integration_tests/pages/checkAnswers.ts

```diff
@@ -38,4 +39,9 @@ export default class CheckAnswersPage extends Page {
     cy.get(this.discardButtonId).click()
     return Page.verifyOnPage(DiscardPage)
   }
+
+  clickBackToCaseload = (): CaseloadPage => {
+    cy.get('.govuk-button-group > .govuk-link').click()
```

Comment:
meh.. good to add data attributes to identify buttons rather than using non-semantic css selectors

`data-qa="back-to-caseload-link"`

Reply:
Just so happened to be reading this on Friday related to this - https://dev.to/christinepinto/data-qa-attribute-a-better-way-to-select-elements-for-ui-test-automation-48lm#:~:text=Solution%3A%20Data%2Dqa%20Attributes&text=These%20HTML%20attributes%20enable%20us,or%20purposes%20on%20the%20page.